tests/pass/stacked-borrows/issue-miri-2389.rs

```diff
@@ -1,3 +1,4 @@
+//@ignore-target-wasm: Enable this when we have a better way of turning off permissive provenance per-test.
```

Comment:
I am confused, what fails here?

Reply:
We have a provenance warning and backtrace in the stderr for this test. We set `MIRIFLAGS="-Zmiri-permissive-provenance"` for all wasm32-wasi tests in ci.sh, and there's no flag to turn warning back *on*. If we run this test under miri, it won't print any warnings to stderr, which will cause a mismatch.

Reply:
Oh I see. Maybe the comment should be "no provenance warning on wasm due to global permissive provenance flag", to make this more clear.